test/integ/routers/alpha-router/alpha-router.integration.test.ts

```diff
@@ -2571,13 +2565,14 @@ describe('quote for other networks', () => {
       c != ChainId.ARBITRUM_GOERLI &&
       c != ChainId.OPTIMISM && /// @dev infura has been having issues with optimism lately
       // Tests are failing https://github.com/Uniswap/smart-order-router/issues/104
-      c != ChainId.CELO_ALFAJORES
+      c != ChainId.CELO_ALFAJORES &&
+      c != ChainId.SEPOLIA
```

Comment:
Why not run tests on sepolia? If there is reasonable liquidity we should run some tests

Reply:
there is not much reasonable liquidity rn :P